### ISSUE TITLE
Reduction of a karma because of change of a role on the accomplice

### ISSUE THREAD

**Issue by dok441** [Alf21/TTT2#64]
Когда вас превращает шакал в sidekick, и вы начинаете убивать, ваша карма будет снижаться и это будет считаться системой кармой как RDM.  Я полагаю, когда шута убивает ясновидящий, ему тоже снимет карму как за rdm.

**Comment by Alf21** [comment]
Please explain in other words and in detail. If can't reproduce this. As far as I know, you get punished for killing your own teammates you were able to know.

**Comment by dok441** [comment]
Нет, я говорю, если невиновного превратят в сообщника и он начнет убивать других невиновных, у него отнимут карму, будто он был невиновным и убивал своих, но его превратили же в сообщника, система кармы этого не понимает.

**Comment by Alf21** [comment]
Hmm... in the translator, the translation has the same result haha :D
Okay i will try to repeat what you tried to say: if you being a sidekick (because of the sidekickdeagle of the jackal e.g.) and you begin to kill ENEMIES, you will lose karma. To be sure, please explain in detail which player of which role you are killing :)

**Comment by dok441** [comment]
Невиновного

**Comment by Alf21** [comment]
Hmm... I need more data, e.g. video or pics. I can't reproduce and still can't find an error in the code. If you kill someone that has your team and if you were able to know him, you will lose karma. If that's not the case (most of all), you won't lose karma.

**Comment by dok441** [comment]
As you will correct weaponshpop, I will surely make detailed screenshots or video.
